### PR TITLE
Explain API authentication when configuring deCONZ

### PR DESCRIPTION
**Pull request by tomaszstrejczek**
## Proposed change

The PR provides details necessary for configuring deCONZ integration. 

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

None.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @tomaszstrejczek, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19060).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/deconz.markdown [hunk 1]
```diff
@@ -60,6 +60,8 @@ See [deCONZ wiki](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/
 
 {% include integrations/config_flow.md %}
 
+Home Assistant connects to Zigbee devices via an API exposed by the deCONZ tool. Therefore, in the process of configuration Home Assistant needs to be authenticated to the tool. This is performed by launching Phoscon App from deCONZ tool, selecting Settings|Gateway|Advanced and clicking "Authenticate app" button.
```
Comment (frenck) [inline source/_integrations/deconz.markdown]: This does not apply to all installation methods. For example, when using the add-on, this procedure would not be correct.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/deconz.markdown (hunk 1 above)
Comment (tomaszstrejczek) [reply source/_integrations/deconz.markdown]: Added further clarification.

File: source/_integrations/deconz.markdown [hunk 2]
```diff
@@ -60,6 +60,8 @@ See [deCONZ wiki](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/
 
 {% include integrations/config_flow.md %}
 
+If manual configuration (without an add-on) is used, then Home Assistant needs to be authenticated to the deCONZ tool. This is performed by launching Phoscon App from deCONZ tool, selecting Settings|Gateway|Advanced and clicking "Authenticate app" button.
```
Comment (Kane610) [inline source/_integrations/deconz.markdown]: Trying to use a wording much closer to the config flow dialogue.
```suggestion
Running a stand-alone instance of deCONZ (non add-on installation) requires a pairing between the deCONZ gateway and Home Assistant. To allow Home Assistant to connect with deCONZ go to the Phoscon UI click Settings -> Gateway -> Advanced and press the "Authenticate app" button. This same information is also shown during the config flow of the deCONZ integration.
```

**Review by Kane610 — APPROVED** [review]
Beautiful

**Review by frenck — APPROVED** [review]
Thanks, @tomaszstrejczek 👍

**Pull request merged**